### ISSUE TITLE
Moderating Topics Created by Unapproved Users

### ISSUE THREAD

**Issue by tigercat** [rubysherpas/forem#474]
If an unapproved user creates a topic, they get a warning from the system that the topic is pending review.  However, when an administrator clicks the "Moderation Tools" button, forem says "No posts".  But there is a post for a new topic created by the unapproved user. 

Looking at the code for the ModerationController:

``` ruby
def index
  @posts = forum.posts.pending_review.topic_not_pending_review
  @topics = forum.topics.pending_review
end
```

One can see `@posts` doesn't include pending topics.  These are stored in `@topics`.

However, the moderation/index view (simplified) is:

``` erb
<%= t('.title', :forum => forum) %>

<%= t('posts_count', :count => @posts.count, :scope => 'forem.general') %>

<%= form_tag forem.forum_moderate_posts_url(forum), :method => :put do %>
  <div id='posts'>
    <% @posts.limit(25).group_by(&:topic).each do |topic, posts| %>
      <%= forem_emojify(topic.forum.title) %> &raquo; <%= forem_emojify(topic.subject) %>
      <%= render posts, :mass_moderation => true %>
    <% end %>
  </div>
  <%= submit_tag "Moderate", :class => "btn btn-primary" %>
<% end %>
```

There is no mention of `@topics`. 

What is the plan here?  Was the moderation/index view meant to include `@topics`?  If we don't display `@topics`, how can administrators easily check for new topics that are pending review?

**Comment by radar** [comment]
Hi @tigercat.

Please remember to format code with triple backticks. This makes it easier for me to see what's code and what's text. I've edited your original post here and you can see what I mean by clicking "Edit" again.

**Comment by radar** [comment]
Yes, the moderation view was supposed to contain topics. This has obviously been overlooked. Would you please be able to submit a patch that adds a list of topics to this view please?

**Comment by tigercat** [comment]
Sorry about the formatting.  Will do it in the future.

I'll look into enhancing the view for topics.

**Comment by radar** [comment]
@tigercat any updates on this?

**Comment by bodrovis** [comment]
<s>I was not able to reproduce this using `rails3` branch. I have two users: an administrator and a ordinary user with `forem_state` set to `pending_review`. I add a post as an ordinary user, then login as an admin, open moderation tools and do see this topic and can approve it. Am I missing something or was it fixed somewhere else?</s>

**Comment by bodrovis** [comment]
Oops sorry I still see "no posts" message.

**Comment by radar** [comment]
Fixed by #545.
